net/dns/manager_windows.go

```diff
@@ -300,6 +300,23 @@ func (m windowsManager) Close() error {
 	return m.SetDNS(OSConfig{})
 }
 
+func (m windowsManager) GetBaseConfig() (OSConfig, error) {
+	if m.nrptWorks {
+		return OSConfig{}, errors.New("GetBaseConfig not supported")
```

Comment:
add `ErrGetBaseConfigNotSupported` to reuse for all impls?

but this path reads work: `if itWorks { fail }` ... add a comment?

Reply:
I guess strictly speaking the windows manager doesn't need to refuse to provide a base config when split-DNS works. Removed that conditional. Added ErrGetBaseConfigNotSupported anyway though.